data/serializer/src/main/java/tech/pegasys/teku/api/schema/BeaconBlock.java

```diff
@@ -30,39 +28,38 @@
   public final UInt64 slot;
 
   @Schema(type = "string", format = "uint64")
-  public final UInt64 proposer_index;
+  public final UInt64 proposerIndex;
```

Comment:
so by doing this, we effectively create some duplication - now the interface isn't sure about whether its `proposer_index` or `proposerIndex`, so we have both.
Need to at least put `@JsonProperty("proposer_index")` to define it. this will also get rid of the typing issue appearing because of the proposer_index type not being given a schema definition.
same goes for all renamed fields here.

Reply:
the other option is leave them named the way the api requires them, given that it's only an object that the api uses...